src/status_im2/contexts/chat/messages/content/text/style.cljs

```diff
@@ -21,7 +21,8 @@
   {:background-color   colors/primary-50-opa-10
    :padding-horizontal 3
    :border-radius      6
-   :margin-bottom      -3})
+   :margin-bottom      -3
```

Comment:
@ibrkhalil I'm not very much sure it will work in Android. Please just check it once!!

Reply:
I posted screenshots from emulator, Will test on my Pixel 7A